.github/workflows/post-apiview.yml

```
@@ -0,0 +1,32 @@
+name: After APIView
+
+on:
+  check_run:
+    types: [completed]
+    
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  post-apiview:
+    name: After APIView
+    runs-on: ubuntu-24.04
+    if: |
+      ${{ github.event.check_run.check_suite.app.name == 'Azure Pipelines' && 
+      ( contains(github.event.check_run.name, 'APIView') ||
+      contains(github.event.check_run.name, '(SDK Generation') ||
+      contains(github.event.check_run.name, 'Azure.azure-rest-api-specs-pipeline') ) }}
```

```suggestion
      github.event.check_run.check_suite.app.name == 'Azure Pipelines' && 
      ( contains(github.event.check_run.name, 'APIView') ||
      contains(github.event.check_run.name, '(SDK Generation') ||
      contains(github.event.check_run.name, 'Azure.azure-rest-api-specs-pipeline'))
```


First, I'm pretty sure that `${{ }}` expressions must be on a single line. Have you tested this yaml? I don't think it will work.

Second, try to align the conditions as I have above, it's easier to read.

---

Am testing on my fork the ${{ }} expression works across multiple lines. https://github.com/chidozieononiwu/azure-sdk-for-net/actions/runs/14139268672

---

To prove the condition is working, can you show me an example where the workflow is skipped?

---

I'm also curious about this question.

---

The one upside to using `${{ }}` I'm aware of, is you get intellisense in VSCode, while you don't with `|`.  There is a feature request to add intellisense for `|` to the VSCode actions plugin, but I'm not expecting it anytime soon.

But even without intellisense, if the check is too long to be read on a single line, I think we have no choice but to use `|` for readability.

---

I have updated the format to multiline.